test/bin/start_webserver.sh

```diff
@@ -9,6 +9,36 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
+usage() {
+    cat - <<EOF
+${BASH_SOURCE[0]} [stop]
```

Comment:
Nit: for consistency, let's rename this script to manage_webserver.sh? 
Also, we need `[start | stop]` or ``[create | cleanup]` arguments for consistency with other cleanup scripts.